### PR TITLE
Fixes broken formatting in the docs

### PR DESCRIPTION
**Pull request by cygnusv**
Fixes broken formatting in the docs index and in the staking guide.

Also, some minor touches here and there wrt to formatting.

### CODE REVIEW

File: docs/source/architecture/contracts.md [hunk 1]
```diff
@@ -82,9 +82,15 @@ The first parameter is used to identify the stake to divide and the last two for
 When calculating locked tokens using the `StakingEscrow.getLockedTokens(address, uint16)` method, all stakes that are active during the specified period are summed.
 
 
+### The Staker Bonds to a Worker ("Ursula")
+The staker must specify a worker who will confirm the activity and sign on behalf of this staker by calling the `StakingEscrow.setWorker(address)` method.
```
Comment (michwill) [inline docs/source/architecture/contracts.md]: Should it be backquotes or double-backquotes?

File: docs/source/architecture/contracts.md [hunk 2]
```diff
@@ -110,10 +116,6 @@ The staker can set a minimum reward rate for a policy. For that, the staker shou
 Some users will have locked but not staked tokens.
 In that case, an instance of the `UserEscrow` contract will hold their tokens (method `UserEscrow.initialDeposit(uint256, uint256)`).
 All tokens will be unlocked after a specified time and the user can retrieve them using the `UserEscrow.withdraw(uint256)` method.
-When the user wants to become a staker - he uses the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.
+When the user wants to become a staker - she uses the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.
```
Comment (michwill) [inline docs/source/architecture/contracts.md]: !

File: docs/source/guides/installation_guide.md [hunk 3]
```diff
@@ -154,8 +154,8 @@ $ ./scripts/installation/install_solc.sh
 
 ## Systemd Service Installation
 
-1. Use this template to create a file named _ursula.service_ and place it in */etc/systemd/system/*.
-
+1. Use this template to create a file named `ursula.service` and place it in `/etc/systemd/system/`.
```
Comment (michwill) [inline docs/source/guides/installation_guide.md]: Same here. Double-backquotes?

File: docs/source/guides/staking_guide.rst [hunk 4]
```diff
@@ -79,7 +79,7 @@ Staking Overview
 
 Most stakers on the Goerli testnet will complete the following steps:
 
-1) Install nucypher on StakeHolder node https://docs.nucypher.com/en/latest/guides/installation_guide.html
+1) Install ``nucypher`` on StakeHolder node https://docs.nucypher.com/en/latest/guides/installation_guide.html
```
Comment (michwill) [inline docs/source/guides/staking_guide.rst]: Also, perhaps, we can make a relative link?

File: docs/source/architecture/contracts.md (hunk 1 above)
Comment (cygnusv) [reply docs/source/architecture/contracts.md]: Single quotes in Markdown, double quotes in RST.

Comment (michwill) [reply docs/source/architecture/contracts.md]: Ah true, that's markdown...

File: docs/source/architecture/contracts.md (hunk 2 above)
Comment (cygnusv) [reply docs/source/architecture/contracts.md]: 👩‍🚀

File: docs/source/architecture/contracts.md [hunk 5]
```diff
@@ -14,6 +14,9 @@
 
 ## Deployment Procedure
 
+This section describes how NuCypher contracts are deployed, step-by-step.
```
Comment (derekpierre) [inline docs/source/architecture/contracts.md]:
```suggestion
This section provides a step-by-step guide of how NuCypher contracts are deployed.
```

File: docs/source/architecture/contracts.md [hunk 6]
```diff
@@ -82,9 +85,15 @@ The first parameter is used to identify the stake to divide and the last two for
 When calculating locked tokens using the `StakingEscrow.getLockedTokens(address, uint16)` method, all stakes that are active during the specified period are summed.
 
 
+### The Staker Bonds to a Worker ("Ursula")
+The staker must specify a worker who will confirm the activity and sign on behalf of this staker by calling the `StakingEscrow.setWorker(address)` method.
+Changing a worker is allowed no more than 1 time in `StakingEscrow.minWorkerPeriods()`.
```
Comment (derekpierre) [inline docs/source/architecture/contracts.md]:
```suggestion
Changing a worker is allowed no more than once within `StakingEscrow.minWorkerPeriods()`.
```

File: docs/source/guides/contribution_guide.rst [hunk 7]
```diff
@@ -21,35 +21,35 @@ Here is the recommended procedure for acquiring the code in preparation for
...
+1. Use GitHub to fork the ``nucypher/nucypher`` repository
 
 2. Clone your fork's repository to your local machine
 
 .. code-block:: bash
 
    $ git clone https://github.com/<YOUR-GITHUB-USERNAME>/nucypher.git
 
-3. Change Directories into ``nucypher``
+3. Change directories into ``nucypher``
```
Comment (derekpierre) [inline docs/source/guides/contribution_guide.rst]:
```suggestion
3. Change directory to ``nucypher``
```

File: docs/source/architecture/contracts.md [hunk 8]
```diff
@@ -110,10 +119,6 @@ The staker can set a minimum reward rate for a policy. For that, the staker shou
 Some users will have locked but not staked tokens.
 In that case, an instance of the `UserEscrow` contract will hold their tokens (method `UserEscrow.initialDeposit(uint256, uint256)`).
 All tokens will be unlocked after a specified time and the user can retrieve them using the `UserEscrow.withdraw(uint256)` method.
-When the user wants to become a staker - he uses the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.
+When the user wants to become a staker - she uses the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.
```
Comment (tuxxy) [inline docs/source/architecture/contracts.md]: :wink: 

```suggestion
When the user wants to become a staker - they use the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.
```

**Pull request merged**